docs/home/whats-new-2-15.md

```diff
@@ -1,94 +1,103 @@
-* Added new functionality to download a report all the data in the table for the following:
-    * Projects
-    * Departments
-    * Nodes
-    * Node pools
-    * Consumption report
+* Added the ability to download a CSV file from all features that contain a table. Downloading a CSV can provide a snapshot of the feature history over the course of time, and help with compliance tracking. All the columns that are selected (displayed) in the table will be downloaded to the file.
 
-Use the *More* menu to download the CSV file. Select from the *Columns* menu to add more columns to the CSV file.
+<!-- RUN-7495/RUN11388 Support PSA / SCCs V2 \(security mechanism for pods on K8S/OCP\)-->
+* Added support for `restricted` policy for [Pod Security Admission](https://kubernetes.io/docs/concepts/security/pod-security-admission/){target=_blank} (PSA) on OpenShift only. For more information, see [Pod security admission](../admin/runai-setup/cluster-setup/cluster-prerequisites.md#pod-security-admission).
 
-## Improvements
+<!-- ADDLINK RUN-10241/RUN-12872 - New Workloads view -->
+* Added a new button to the *Jobs* feature to switch the view to *Workloads*. *Workloads* is a new view for jobs that are running in the platform. The *Workloads* view provides a more advanced UI than the previous *Jobs* UI. The new table format provides:
 
-<!-- TODO RUN-9943/RUN-12176 Nodes - reflect the correct status of the node - add to nodes page the table from the TW ticket -->
-* Improved the readability of the node table to include a more detailed status and its description. For more information, see [Page here](page here)
+      * Improved views of the data
+      * Improved filters and search
+      * More information
 
-<!-- RUN-11421 Consumption report - Cost and bugs-->
-* Improved the Consumption report interface by moving the Cost settings to the *General* settings menu.
+    For more information see [Workloads](##workloads-view).
 
-<!-- RUN-5928 Node Scheduler - MS1 -->
+<!-- RUN-12313/12314 - CPU focused dashboards -->
+* Added a new dashboard for CPU based environments. The dashboards display specific information for CPU based nodes, node-pools, clusters, or tenants. These dashboards also include additional metrics that specific to CPU based environments. This will help optimize visual information eliminating the views of empty GPU dashlets. For more information see [CPU Dashboard](../admin/admin-ui-setup/dashboard-analysis.md#cpu-dashboard).
 
-<!-- RUN-7085/RUN-9480 Installation - Cluster wizard Improvements -->
+<!-- RUN-10622/RUN-10625 Policy blocks workloads that attempt to store data on the node-->
+* Added the ability to prevent the submission of workloads that use data sources that consists of a host path using policies. This is prevents data from being stored on the node. When a node is deleted, all data stored on that node is lost. For configuration information, see [Prevent Data Storage on the Node](##prevent-data-storage-on-the-node).
```

Comment:
This is prevents?

Reply:
Added the ability to prevent the submission of workloads that use data sources of **type** host path using policies. This prevents data from being stored on the node. When a node is deleted, all data stored on that node is lost. For configuration information, see [Prevent Data Storage on the Node](##prevent-data-storage-on-the-node